eng/vmr-sync.sh

```diff
@@ -180,7 +200,7 @@ fi
 highlight "Starting the synchronization to '$target_ref'.."
 set +e
 
-if "$dotnet" darc vmr update --vmr "$vmr_dir" --tmp "$tmp_dir" --$verbosity --recursive --additional-remotes "installer:$installer_dir" "installer:$target_ref"; then
+if "$dotnet" darc vmr update --vmr "$vmr_dir" --tmp "$tmp_dir" --$verbosity --recursive --readme-template $readme_template --tpn-template $tpn_template --additional-remotes "installer:$installer_dir" "installer:$target_ref"; then
```

Comment:
```suggestion
if "$dotnet" darc vmr update --vmr "$vmr_dir" --tmp "$tmp_dir" --$verbosity --recursive --readme-template "$readme_template" --tpn-template "$tpn_template" --additional-remotes "installer:$installer_dir" "installer:$target_ref"; then
```

Reply:
off-topic: I suggest you install VS Code extension ShellCheck that will give you hints about situations like these